github/ci/services/prometheus-stack/manifests/stack/common/kube-prometheus-stack.yaml

```diff
@@ -237,7 +237,7 @@ imagePullSecrets:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: prometheus-stack-kube-prom-prometheus
+  name: prometheus-k8s
```

Comment:
This can be disruptive for the existing deployed stacks, can't we keep it as is for them and only apply the new name in the performance cluster?

Reply:
I am using patches to replace the service account name only in the performance cluster